Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.22 as builder
```

Comment:
Let's keep 1.22 for now. All the code changes will get backported to 4.18 which still needs 1.22.